lib/messages.h

```diff
@@ -80,7 +81,7 @@ void msg_add_option_group(GOptionContext *ctx);
 
 #define msg_debug(desc, tag1, tags...) 						  \
 	do { 									  \
-	  if (G_UNLIKELY(debug_flag))             				  \
+	  if (G_UNLIKELY(debug_flag) || G_UNLIKELY(startup_debug_flag))             				  \
 	    msg_event_suppress_recursions_and_send(                               \
```

Comment:
G_UNLIKELY should encapsulate both conditions.

The backslash should be aligned with the rest in the macro.